userdocs/src/usage/eks-managed-nodes.md

```diff
@@ -246,8 +246,6 @@ eksctl scale nodegroup --name=managed-ng-1 --cluster=managed-cluster --nodes=4 -
 EKS Managed Nodegroups are managed by AWS EKS and do not offer the same level of configuration as unmanaged nodegroups.
 The unsupported options are noted below.
 
-- Tags (managedNodeGroups[*].tags) in managed nodegroups apply to the EKS Nodegroup resource and to the EC2 instances launched as part of the nodegroup.
```

Comment:
I wouldn't completely delete this sentence as they still propagate. Just change the last entry to say, `and to the Autoscaling Group.`

Reply:
I think this is deleted because it is in the "unsupported options" aka "limitations" section - which with this PR it no longer is

Reply:
@afirth answered to this https://github.com/weaveworks/eksctl/pull/5002#discussion_r841743039

There not no longer limitations about it: cf "The unsupported options are noted below."

Do you still want me to re-add it?

Reply:
Nah, it's fine. :)